Documentation/install/upgrade.rst

```diff
@@ -265,6 +265,13 @@ Annotations:
 1.5 Upgrade Notes
 -----------------
 
+New Default Values
+~~~~~~~~~~~~~~~~~~
+
+ * The connection-tracking garbage collector intervals is now 12 hours when
+   using LRU maps (newer kernels) and 15 minutes an all older kernels. The
```

Comment:
how newer? please specify the versions

Reply:
I don't like this because many distributions will backport so the version may or may not be meaningful. I'm not sure it adds much value in the upgrade notes.